installer/scripts/omsadmin.sh

```diff
@@ -787,6 +804,40 @@ show_workspace_status()
     fi
 }
 
+show_omsagent_proc_limit()
+{
+    local result_limit=
+
+    if [ ! -f $PROC_LIMIT_CONF ]; then
+        log_error "Limits configuration file '$PROC_LIMIT_CONF' missing!"
+        return 1
+    fi
+    local count=`cat $PROC_LIMIT_CONF | grep -E "$LIMIT_LINE_REGEX" | wc -l`
```

Comment:
I just remembered; there is already a helper function [find_current_omsagent_proc_limit](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/scripts/omsadmin.sh#L405) which cats the file and greps it, but does not count the number of lines. Please use that function in place of all of the cat/grep lines.

Reply:
Okay, so this is in my latest branch check in.  I haven't squashed yet, but tell me if you think we are getting close, then I will.